packages/react-component-library/src/components/Timeline/TimelineEvents.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import { TimelineEventProps } from '.'
+
+export interface TimelineEventsProps extends ComponentWithClass {
+  children:
+    | React.ReactElement<TimelineEventProps>
+    | React.ReactElement<TimelineEventProps>[]
+}
+
+export const TimelineEvents: React.FC<TimelineEventsProps> = (props) => {
+  return (
+    <div {...props} />
```

Comment:
Could we possibly show the children explicitly being drilled?

It confused me initially not seeing it a bit like this:

```jsx
<div {...rest}>
  {children}
</div>
```